index.js

```diff
@@ -48,15 +48,14 @@ const query = `
 let responseText = '[]';
 
 async function fetchRepos() {
-	const {body} = await got.post('https://api.github.com/graphql', {
-		json: true,
-		body: {query},
+	const {body} = await graphqlGot('https://api.github.com/graphql', {
+		query,
 		headers: {
 			authorization: `bearer ${token}`
 		}
```

Comment:
Should it handle `token` too? Like `gh-got`? I assume most GraphQL endpoints require a token.

Reply:
Good idea!

Reply:
Feature added now :).